frontend/public/images/icons/youtube.svg

```diff
@@ -0,0 +1,11 @@
+<svg xmlns="http://www.w3.org/2000/svg" width="60" height="60" viewBox="0 0 60 60">
+  <g id="Group_437" data-name="Group 437" transform="translate(-414 -592)">
+    <g id="Ellipse_5" data-name="Ellipse 5" transform="translate(414 592)" fill="none" stroke="#fff" stroke-width="2">
+      <circle cx="30" cy="30" r="30" stroke="none"/>
+      <circle cx="30" cy="30" r="29" fill="none"/>
+    </g>
+    <g id="youtube-icon" transform="translate(429 537.912)">
+      <path id="Path_259" data-name="Path 259" d="M29.983,81.291q-.016-.9-.142-2.285a20.784,20.784,0,0,0-.36-2.469,3.893,3.893,0,0,0-1.163-2.059,3.568,3.568,0,0,0-2.084-.971A109.83,109.83,0,0,0,15,73.088a109.826,109.826,0,0,0-11.233.418,3.53,3.53,0,0,0-2.076.971A3.915,3.915,0,0,0,.536,76.537a18.529,18.529,0,0,0-.377,2.469Q.034,80.387.017,81.291T0,83.8q0,1.607.017,2.511T.159,88.6a20.707,20.707,0,0,0,.36,2.47,3.891,3.891,0,0,0,1.163,2.059,3.57,3.57,0,0,0,2.084.971A109.72,109.72,0,0,0,15,94.517,109.721,109.721,0,0,0,26.233,94.1a3.53,3.53,0,0,0,2.076-.971,3.916,3.916,0,0,0,1.155-2.059,18.541,18.541,0,0,0,.377-2.47q.125-1.381.142-2.285T30,83.8Q30,82.2,29.983,81.291Zm-9.057,3.415-8.571,5.357a.955.955,0,0,1-.569.167,1.158,1.158,0,0,1-.519-.134,1,1,0,0,1-.552-.937V78.445a1,1,0,0,1,.552-.937,1.008,1.008,0,0,1,1.088.033L20.926,82.9a1.065,1.065,0,0,1,0,1.808Z" fill="#fff"/>
+    </g>
+  </g>
+</svg>
```

Comment:
Aren't we deleting the original icon SVGs after converting them to React components?